src/main/jbake/content/docs/user/getting-started.adoc

```diff
@@ -81,10 +81,39 @@ Status: Image is up to date for hawkular/hawkular:latest #<2>
 <1> We ask for the latest build
 <2> In this case the local one was already the latest, so we are good here.
 
-
 === Working on Hawkular source
 
 If you are interested in hacking on Hawkular, or building it from source, then check out
 link:/docs/dev/development.html[Development resources]
 
+== Install a Wildfly monitoring agent
+
+If you wish to monitor a WildFly instance,  you can do so with the following steps:
+
+. Download the WildFly module from http://download.jboss.org/hawkular/wildfly-monitor/0.0.2/hawkular-monitor-0.0.2-module.zip[here].
+. Unzip the file inside the `modules` directory of  your WildFly 8.2 instance
+. Add the following subsystem declaration in the `standalone/configuration/standalone.xml` file of your WildFly instance, the `url` of the storage adapter should be the URL of your Hawkular server.
+
+[source,xml]
+----
+ <subsystem enabled="true" apiJndiName="java:global/hawkular/agent/monitor/api" numMetricSchedulerThreads="3" numAvailSchedulerThreads="3" xmlns="urn:org.hawkular.agent.monitor:monitor:1.0">
+            <storage-adapter type="HAWKULAR" url="http://localhost:8080" context="/hawkular-bus/message/" restContext="/hawkular-metrics/"/>
```

Comment:
The '<storage-adapter>' syntax has changed.

First, there is no longer "context" and "restContext" attributes but rather optional contexts for the different server-side components (inventoryContext, metricsContext, etc). Because we don't support changing context names in our server-side components today, just leave them out. The defaults will be fine.
Second, you'll need to provide credentials that are used by the agent to sign into Hawkular. When you create a user in hawkular-accounts, you tell the agent via the "username" and "password" attributes. You can use the WildFly vault to obfuscate the password if you want (though I haven't tested it - I assume WildFly will decrypt the password before passing it on to the agent service).

Third, the URL attribute is still supported, however, it is no longer the "official" way to set it. Now we support outbound-socket-bindings (and we recommend using this instead since this is the "official" way Wildfly configures outbound ports).  So, to point to a hawkular instance, you would define a outbound-socket-binding in your socket-binding-group (just as out-of-box WildFly defines an outbound-socket-binding for SMTP server) and tell the agent what socket binding to use. For example:

'<storage-adapter type="HAWKULAR" username="SET_ME" password="SET_ME" serverOutboundSocketBindingRef="hawkular"/>'

At the bottom of standalone.xml, you would then add this to your <socket-binding-group>:

'<outbound-socket-binding name="hawkular">
   <remote-destination host="your-hawkular-server" port="8080" />
</outbound-socket-binding>'